src/Repository/Model/Catalog.php

```diff
@@ -2933,10 +2933,10 @@ public static function process_action($action, $catalogs, $options = null)
                 $catalogs = self::get_catalogs();
                 // Intentional break fall-through
             case 'update_file_tags':
-                $write_id3 = AmpConfig::get('write_id3', false);
-                AmpConfig::set('write_id3', 'true', true);
+                $write_id3     = AmpConfig::get('write_id3', false);
                 $write_id3_art = AmpConfig::get('write_id3_art', false);
-                AmpConfig::set('write_id3_art', 'true', true);
+                AmpConfig::set_by_array(['write_id3' => 'true'], true);
+                AmpConfig::set_by_array(['write_id3_art' => 'true'], true);
```

Comment:
After merging this, I'll check if it's possible to drop those temporary config settings and replace them by some parameters for the tagwriter